spine/simpleNativeConnection.js

```diff
@@ -1,157 +1,174 @@
-'use strict';
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';  // We need this for testing with self signed certs
 
-var XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest,
-    nodeUrl = require ('url'),
-    Erizo = require('./erizofc'),
-    NativeStream = require ('./NativeStream.js'),
-    NativeStack = require ('./NativeStack.js');
+const XMLHttpRequest = require('xmlhttprequest').XMLHttpRequest; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
We should, i'd do it when we create a package.json for each of the parts of licode.